server/controllers/inventory/reports/prices.handlebars

```diff
@@ -30,22 +33,25 @@
                 {{ name }}
               </th>
 
-              <th colspan="3" style="border:none; border-bottom: solid black 2px;" class="text-right">
+              <th colspan="6" style="border:none; border-bottom: solid black 2px;" class="text-right">
```

Comment:
Could you make the `{{ name }}` `<th>` colspan="3" and this one colspan="3"?  That way the name won't be squished so much by the extra columns added here.  See below:
![medicinesandcomprimes](https://user-images.githubusercontent.com/896472/33614430-abcc1244-d9d7-11e7-976c-8a3d852048c4.PNG)
_colspan=3 would bring the name's `<td>` out to the red line_